src/components/Home.js

```diff
@@ -0,0 +1,8 @@
+import React from'react'
+const Home =() => {
```

Comment:
curious about why you use this syntax here and `function About() {` in the About.js file above.  As I understand it, they are both achieving the same thing.